tests/test_models.py

```diff
@@ -82,14 +82,20 @@ def test_model_singlepsr_noise_dip_cusp(nodmx_psrs,caplog):
     assert hasattr(m,'get_lnlikelihood')
     x0 = {pname:p.sample() for pname,p in zip(m.param_names, m.params)}
     m.get_lnlikelihood(x0)
-
+    
```

Comment:
This is line has four space and notion else.  This type of thing is called a "whitespace error".  Can you delete the spaces, so it's a truly "empty line"